### PR TITLE
Add Events#addEmitter and Events#removeEmitter 

### PR DESCRIPTION
**Pull request by vkalpias**
Add Events#addEmitter and Events#removeEmitter to allow proxying events from this emitter to additional ones.

This will be used for example in the Editor API to emit the same events from an Observer to its container class.

### CODE REVIEW

File: src/binding/events.js [hunk 1]
```diff
@@ -88,23 +90,29 @@ Events.prototype.emit = function (name, arg0, arg1, arg2, arg3, arg4, arg5, arg6
...
+                events[i].call(this, arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7);
+            } catch (ex) {
+                console.info('%c%s %c(event error)', 'color: #06f', name, 'color: #f00');
+                console.log(ex.stack);
+            }
         }
     }
 
+    if (this._additionalEmitters.length) {
+        const emitters = this._additionalEmitters.slice();
```
Comment (yaustar) [inline src/binding/events.js:110]: Should this match line 94 and be `this._additionalEmitters.slice(0);`?

**Review by mvaligursky — APPROVED** [review]

File: src/binding/events.js [hunk 2]
```diff
@@ -88,23 +90,29 @@ Events.prototype.emit = function (name, arg0, arg1, arg2, arg3, arg4, arg5, arg6
...
-        if (! events[i])
-            continue;
-
-        try {
-            events[i].call(this, arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7);
-        } catch (ex) {
-            console.info('%c%s %c(event error)', 'color: #06f', name, 'color: #f00');
-            console.log(ex.stack);
+    if (events) {
+        events = events.slice(0);
```
Comment (yaustar) [inline src/binding/events.js:94]: Should we check length here as well?

**Review by yaustar — APPROVED** [review]

File: src/binding/events.js (hunk 1 above)
Comment (vkalpias) [reply src/binding/events.js:110]: don't think it matters

File: src/binding/events.js (hunk 2 above)
Comment (vkalpias) [reply src/binding/events.js:94]: added

**Pull request merged**